conventional_pre_commit/hook.py

```diff
@@ -28,6 +28,7 @@ def main(argv=[]):
         action="store_true",
         help="Force commit to strictly follow Conventional Commits formatting. Disallows fixup! style commits.",
     )
+    parser.add_argument("scopes", type=str, nargs="*", default=None, help="Optional list of scopes to support")
```

Comment:
Adding the argument as an inline `nargs="*"` works for command-line usage of the tool, like:

```
conventional-pre-commit "feat(api): made a change" api
```

However, for the primary use-case as a `pre-commit` hook, we can't pass any arguments _after_ the path to the commit message file, as `pre-commit` passes this in for us automatically. See more at https://pre-commit.com/#passing-arguments-to-hooks

> You can pass static arguments by specifying the [`args`](https://pre-commit.com/#config-args) property in your `.pre-commit-config.yaml`
> ...
> If you are writing your own custom hook, your hook should expect to receive the `args` value and then a list of staged files.

So we need this argument to be a flag-style like `--scopes` that can be used from the `.pre-commit-config.yaml` file.

Reply:
This should be fixed by commit 22d377a66f31d847b174c1810964a42b9fc84b2d.